src/components/JsonSchemaValidator/Schemas/dbtAttribution_0.1.0.js

```diff
@@ -0,0 +1,228 @@
+export const Schema = {
+  "definitions": {
+    passthrough_vars: {
+      type: 'array',
+      description: '> Click the plus sign to add a new entry',
+      minItems: 0,
+      items: {
+        title: "Type",
+        oneOf: [
+          {
+            type: 'string',
+            title: "Column Name"
+          },
+          {
+            type: 'object',
+            title: "SQL & Alias",
+            properties: {
+              sql: { type: 'string' },
+              alias: { type: 'string' } // TODO: add regex here to make valid SQL name?
+            },
+            required: ['sql', 'alias'],
+            additionalProperties: false
+          }
+        ]
+      },
+      uniqueItems: true,
+    }
+  },
+  type: 'object',
+  properties: {
+    snowplow__conversions_source: {
+      type: 'string',
+      title: 'Conversions Source',
+      description: 'Source of conversion events',
+      longDescription: 'The source (schema and table) of your conversion events, likely your atomic events table.',
+      packageDefault: "{{ source('atomic', 'events') }}",
+      group: 'Warehouse and Tracker',
+    },
+    snowplow__conversion_path_source: {
+      type: 'string',
+      title: 'Conversion Path Source',
+      description: 'Source of paths (touchpoints) table',
+      longDescription: 'The source (schema and table) of the  paths (touchpoints). By default it is the derived `snowplow_unified_views` table.',
+      packageDefault: "{{ source('derived', 'snowplow_unified_views') }}",
+      group: 'Warehouse and Tracker',
+    },
+    snowplow__conversions_source_filter: {
+      type: 'string',
+      title: 'Conversions Source Filter',
+      group: 'Operation and Logic',
+      longDescription: ' A timestamp field the conversion source field is partitioned on (ideally) for optimized filtering, when left blank `derived_tstamp` is used.',
+      packageDefault: 'blank',
+    },
```

Comment:
The current default is `cv_tstamp`?

Reply:
this is not needed anymore, all is done within the conversions_clause, I'll get this removed from the package as well